src/content/docs/apm/agents/python-agent/python-agent-api/addcustomattributes-python-agent-api.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: add_custom_attributes (Python agent API)
+type: apiDoc
+shortDescription: Adds one or more custom attributes to a transaction.
+tags:
+  - Agents
+  - Python agent
+  - Python agent API
+metaDescription: 'Python API: This call adds a custom attribute (key/value tuple) to a transaction.'
+redirects:
+  - /docs/agents/python-agent/python-agent-api/addcustomattributes-python-agent-api
+---
+
+## Syntax
+
+```
+newrelic.agent.add_custom_attributes(items)
+```
+
+Adds one or more custom attributes to a transaction.
```

Comment:
That's a great question!  This function just calls the singular function multiple times.  I changed it to say "multiple custom attributes" so that it is hopefully less confusing (even though you could technically add a single one with this one)